### PR TITLE
ci: Fix failing release due to unsupported dart version

### PR DESCRIPTION
**Pull request by mbfakourii**
## Pull Request

- Report security issues [confidentially](https://github.com/parse-community/Parse-SDK-Flutter/security/policy).
- Any contribution is under this [license](https://github.com/parse-community/Parse-SDK-Flutter/blob/master/LICENSE).
- Link this pull request to an [issue](https://github.com/parse-community/Parse-SDK-Flutter/issues?q=is%3Aissue).

## Issue

Closes: #1028

## Approach
n/a

## Tasks

- [ ] Add tests
- [ ] Add changes to documentation (guides, repository pages, code comments)

### CODE REVIEW

File: .github/workflows/release-automated.yml [hunk 1]
```diff
@@ -26,13 +26,13 @@ jobs:
...
         uses: dart-lang/setup-dart@v1
         with:
-          sdk: '3.0'
+          sdk: '3.2.6'
       - name: Setup flutter
         if: env.package == 'flutter'
         uses: subosito/flutter-action@v2
         with:
-          channel: 'stable'
-          cache: true
```
Comment (mtrezza) [inline .github/workflows/release-automated.yml:35]: Why was cache: true removed?

Comment (mbfakourii) [reply .github/workflows/release-automated.yml:35]: The reason for using cache was not clear to me.
Is there a specific reason?

Comment (mtrezza) [reply .github/workflows/release-automated.yml:35]: It speeds up the CI

File: .github/workflows/release-automated.yml [hunk 2]
```diff
@@ -26,13 +26,13 @@ jobs:
...
+          channel: stable
```
Comment (mtrezza) [inline .github/workflows/release-automated.yml:34]:
```suggestion
          cache: true
          channel: stable
```

**Review by mtrezza — APPROVED** [review]

**Pull request merged**